### PR TITLE
Fix build and segment fault on android

### PR DESCRIPTION
**Pull request by veronicaz41**
Fix get_device_list segfault on android 64bit.

### CODE REVIEW

File: android/jni/examples.mk [hunk 1]
```diff
@@ -18,7 +18,7 @@
 
 LOCAL_PATH:= $(call my-dir)
 LIBUSB_ROOT_REL:= ../..
-LIBUSB_ROOT_ABS:= $(LOCAL_PATH)/../..
```
Comment (jdonald) [inline android/jni/examples.mk]: Should this just be `$(LOCAL_PATH)/..`? But then `LIBUSB_ROOT_REL` would just be `..`

Comment (veronicaz41) [reply android/jni/examples.mk]: fixed

**Comment by jdonald** [comment]
Thanks for fixing `ndk-build`. Nice that it builds mips64, x86, and all now.

**Review by jdonald — APPROVED** [review]

**Comment by jdonald** [comment]
Asked Veronica about what we did regarding `-llog`: answer is that was added in `upgrade_tool`'s build flow.

File: libusb/core.c [hunk 2]
```diff
@@ -682,6 +682,10 @@ ssize_t API_EXPORTED libusb_get_device_list(libusb_context *ctx,
 		usbi_mutex_unlock(&ctx->usb_devs_lock);
 	} else {
 		/* backend does not provide hotplug support */
+		if (!usbi_backend->get_device_list) {
+	          len = -1;
+		  goto out;
+		}
```
Comment (jdonald) [inline libusb/core.c:688]: I didn't test this on all of Mac/Linux, but pretty sure it's a safe change as we aren't using `lib{usb,uvc}_get_device_list()`  currently.

**Pull request merged**